src/odemis/gui/cont/views.py

```diff
@@ -97,6 +97,10 @@ def _create_views_fixed(self, viewports):
         visible_views = []
 
         for vp, vkwargs in viewports.items():
+
+            if vkwargs['name'] is None:
```

Comment:
I can't see when this is going to make a difference, as there doesn't seem any place which sets `["name"]` to `None`. So, if it's some left-over from a trial, just remove this code. If it's useful, add the right information in the docstring of `ViewPortController`.